pom.xml

```diff
@@ -91,9 +91,9 @@
     <quicheHomeDir>${project.build.directory}/quiche</quicheHomeDir>
     <quicheHomeBuildDir>${quicheHomeDir}/build</quicheHomeBuildDir>
     <quicheHomeIncludeDir>${quicheHomeDir}/include</quicheHomeIncludeDir>
-    <quicheRepository>https://github.com/cloudflare/quiche</quicheRepository>
-    <quicheBranch>master</quicheBranch>
-    <quicheCommitSha>0a08fd78134dc35351ce773f143015ed12f53ae1</quicheCommitSha>
+    <quicheRepository>https://github.com/normanmaurer/quiche</quicheRepository>
+    <quicheBranch>ids</quicheBranch>
+    <quicheCommitSha>9cd9a00673b9fc13c12529526895b393e23e0d49</quicheCommitSha>
```

Comment:
We need to wait till the PR is merged into quiche so we can not depend on a fork